RecoTracker/DeDx/plugins/DeDxEstimatorProducer.h

```diff
@@ -20,6 +20,7 @@
 #include "DataFormats/TrackReco/interface/DeDxData.h"
 #include "DataFormats/TrackReco/interface/TrackDeDxHits.h"
 #include "DataFormats/TrackReco/interface/DeDxHit.h"
+#include "DataFormats/TrackReco/interface/TrackDeDxHits.h"
```

Comment:
```suggestion
```
(duplicated)

Reply:
Addressed in 05c2dc85edef1e8e3a387cd8694941aa4c4e24bd